index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@200&family=Roboto:wght@300&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header id="sub-container">
+      <nav class="navbar">
+        <a href="#" class="menu-icon">
+          <img src="./images/my-logo/logo3.jpg" alt="logo" width="35"
+        /></a>
+        <div class="menu">
+          <img src="./images/menu/Icon-Menu.png" alt="menu icon" />
```

Comment:
- [ ] Please kindly ensure that you implement the changes requested by the previous reviewer to make your project more professional like wrapping your menu icon in `a` tag. you got this 👍